notebooks/metadata-extraction.ipynb

```diff
@@ -180,7 +193,7 @@
    "source": [
     "for d in sample_docs:\n",
     "    print(f\"Path: {d.properties['path']}\")\n",
-    "    for k in [\"dateAndTime\", \"day\", \"aircraft\", \"location\"]:\n",
+    "    for k in [\"dateTime\", \"day\", \"aircraft\", \"location\"]:\n",
```

Comment:
Are these stable?  I remember Vinayak mentioning at one point that they aren't.  dateAndTime is mentioned above in convert_timestamp, but if one is missing and the other is present you could add code to normalize to dateTime.

Reply:
This is working, It was stable for multiple test runs I did.